deploy/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources: 
+- crds/k8s_v1alpha1_redis_crd.yaml
```

Comment:
I've seen with other Operator implementations two behaviors different to the one used by this Operator:

1. The Operator does inject the CRDs itself at start up
2. The Operator does not inject the CRDs itself at start up, but you have to do manually. But you can only apply the CRD YAML once the Operator has started up and declared that it supports this CRD.

Are you sure that applying the CRD before having the Operator running will work?

Reply:
Shouldn't be a problem. I've created helm chart for this operator, which installs CRDs before applying any other resources, works like a charm.